data/strings/types_strings.txt

```diff
@@ -20567,7 +20309,7 @@
     zh-Hant = 汽車零件
 
   [type.shop.car_repair]
-    en = Car Repair Shop
+    en = Car Repair
```

Comment:
Correct probably, but people in the US say "My car is in the shop".  But it's sort of short for workshop.

Reply:
changed to Car Repair Workshop